README.md

```diff
@@ -138,12 +134,34 @@ _Additional information_
 * `field_delimiter` (`default=none`)
   * Custom field delimiter, for when format is set to `TEXTFILE`
 * `table_properties`: table properties to add to the table, valid for Iceberg only
-* `lf_tags` (`default=none`)
-  * lf tags to associate with the table
-  * format: `{"tag1": "value1", "tag2": "value2"}`
-* `lf_tags_columns` (`default=none`)
-  * lf tags to associate with the table columns
-  * format: `{"tag1": {"value1": ["column1": "column2"]}}`
+* `lf_tags_config` (`default=none`)
+  * [AWS lakeformation](#aws-lakeformation-integration) tags to associate with the table and columns
+  * format for model config:
+```json
+{
+  "enabled": true,
+  "tags": {
+    "tag1": "value1",
+    "tag2": "value2"
+  },
+  "tags_columns": {
+    "tag1": {
+      "value1": ["column1", "column2"]
+    }
+  }
+}
+```
+* format for `dbt_project.yml`:
+```yaml
+  +lf_tags_config:
```

Comment:
this can be set also per model level in the config right? maybe you can provide also an example for it?

Reply:
I've added it in above section

Reply:
That't not what I meant, you added the lf_tags_config the section of supported parameter.
Will be good also to have something like this
```
{{ config(
    materialized='incremental',
    table_type='iceberg',
    incremental_strategy='merge',
    unique_key='user_id',
    lf_tags_config={'enabled': true, 'tags': {'tag1': 'x', 'tag2': 'y'}, 'tags_columns': '....
    format='parquet'
) }}
```

Also we might want to document this better in the new https://github.com/dbt-athena/dbt-athena.github.io, in there we can even create a total new section for lakeformation.

Reply:
I've updated config, now it seems more clear. And yes, once this PR is merged, I'll create separate one for docs site